pvlib/iotools/psm3.py

```diff
@@ -165,6 +165,14 @@ def get_psm3(latitude, longitude, api_key, email, names='tmy', interval=60,
     attributes = [amap.get(a, a) for a in attributes]
     attributes = list(set(attributes))  # remove duplicate values
 
+    if (leap_day is None) & (not names.startswith('t')):
```

Comment:
Nitpick: `and` and `&` have identical semantics when applied to booleans as far as I know (except short-circuiting), but stylistically I prefer to reserve `&` for situations where it actually behaves differently (e.g. element-wise array logic or bit fiddling).  But I don't think PEP8 opines on this point, and from how often I see people using `&` for scalar booleans I guess a lot of people would disagree.

Reply:
Sure, why not :grinning: